src/test/java/network/brightspots/rcv/TabulatorTests.java

```diff
@@ -193,6 +194,22 @@ static void setup() {
     Logger.setup();
   }
 
+  @Test
+  @DisplayName("test isVersionNewer")
+  void testVersionComparison() {
+    // Basic checks: version numbers work fine
+    assertFalse(isVersionNewer("1.3.0", "1.3.0"));
+    assertFalse(isVersionNewer("1.2.0", "1.3.0"));
+    assertFalse(isVersionNewer("1.3.0", "1.3.1"));
+    assertTrue(isVersionNewer("1.3.1", "1.3.0"));
+
+    // Version comparison ignores the -alpha specifier
+    assertFalse(isVersionNewer("1.2.0-alpha", "1.3.0"));
+    assertTrue(isVersionNewer("1.4.0-alpha", "1.3.0"));
+    assertFalse(isVersionNewer("1.4.0-alpha", "1.4.0"));
+    assertFalse(isVersionNewer("1.4.0", "1.4.0-alpha"));
+  }
```

Comment:
this isn't really a tabulator test -- is there anywhere else unit tests might go? Is here okay or should I break out a second file?

Reply:
I'm cool with kicking off a `ContestConfigMigrationTests` class and putting this in there if it's not too much trouble.